src/main/kotlin/org/rust/ide/refactoring/extractFunction/RsExtractFunctionHandler.kt

```diff
@@ -75,6 +71,27 @@ class RsExtractFunctionHandler : RefactoringActionHandler {
         }
     }
 
+    private fun findExistingInherentImpl(traitImpl: RsImplItem): RsImplItem? {
```

Comment:
I'd add a descriptive comment about what this method does. For example, it might be not obvious whether it looks into `impl` blocks inside other items or only checks top-level `impl` blocks